doc/static/css/nrf.css

```diff
@@ -1,43 +1,6 @@
-/* background color top and bottom left */
-
-.wy-side-nav-search {
- background-color: #00a9ce;
-}
-
-.rst-versions .rst-current-version {
- background-color: #00a9ce;
-}
-
-/* Zephyr background color for the doc switcher */
-.rst-versions {
- background: #a67dfb;
-}
-
-
-/* text color navigation menu */
-
-/* l1 not selected */
-.wy-menu-vertical a {
- color: #00a9ce;
-}
-/* expanded container nodes */
-.wy-menu-vertical li.on a, .wy-menu-vertical li.current > a {
- color: #00a9ce;
-}
-/* expanded nodes, not selected */
-.wy-menu-vertical li.current a {
- color: #00a9ce;
-}
-
-
-/* link text */
-.wy-nav-content a {
- color: #00a9ce;
+:root {
+  --docset-color: #00a9ce;
 }
-.wy-nav-content a:visited {
- color: #0077c8;
```

Comment:
Visited link color is now missing (that could be set for all docsets).

Reply:
fixed

Reply:
note that this behavior is not available for all docsets now, so I prefer to do style changes afterwards